Makefile

```diff
@@ -316,6 +316,9 @@ test/%: test/%.chpl
 print-%:
 	@echo "$($*)"
 
+test-python: 
+	pytest -c pytest.ini
```

Comment:
We have a number of systems where python2 is the default. Maybe `python3 -m pytest -c pytest.ini` to ensure 3 is being used?

Reply:
Sure, I can make that change

Reply:
Still game for this change?